cypress/integration/2.3-checkout_scenarios.spec.js

```diff
@@ -0,0 +1,53 @@
+import {
+  testSetup,
+  updateRetry,
+  preserveCookie,
+} from '../support/common/support.js'
+import { checkoutScenario } from '../support/outputvalidation.js'
+import selectors from '../support/common/selectors.js'
+
+describe('Testing Checkout with different scenarios', () => {
+  // Load test setup
+  testSetup()
+
+  const { prefix, productName, productPrice } = checkoutScenario
+
+  it(`In ${prefix} - Adding product,remove product and add product again`, () => {
+    // Search the product
+    cy.searchProduct(productName)
+    // Add product to cart
+    cy.addProduct(productName)
+    // Remove the product
+    cy.removeProduct('880300')
+    // Search the product
+    cy.searchProduct(productName)
+    // Add product to cart
+    cy.addProduct(productName)
+  })
+
+  it(`In ${prefix} Updating product quantity to 2`, updateRetry(3), () => {
```

Comment:
It's really necessary update quantity again? I saw it on the previous test.

Reply:
yes it is a testcase where we check if quantity gets increased total gets increased/ if quantity gets decreased the total also gets decreased

Reply:
But do it one time in one test isn't enough to assure it is working? You are doing it three times... and update/decrease quantity is a flaky test as we can see on Cypress Dashboard.